webapp/app/static/css/components.css

```diff
@@ -186,42 +186,46 @@
 }
 
 /* DETAILS */
-.details-card .card{
+.details-card .card {
     border: 0;
     border-radius: 0;
     background: inherit;
 }
 
-.details-card .card-header{
+.details-card .card-header {
     border: 0;
     background: inherit;
 }
 
-.details-card .card-header h5{
+.details-card .card-header span {
     color: var(--text-color);
     font-size: var(--text-medium);
     font-weight: var( --font-bold);
 }
 
-.details-card .card-header button{
+.details-card .card-header button {
     text-align: left;
-    color: var(--text-colour);
+    color: var(--text-color);
     background: inherit;
     border: 0;
 }
 
-.details-card .card-header .collapsed h5{
+.details-card .card-header .collapsed span {
     color: var(--link-color);
     text-decoration: underline;
     text-decoration-color: var(--icon-default-colour);
 }
 
-.details-card .card-header .collapsed h5:hover{
+.details-card .card-header .collapsed span:hover {
     color: var(--link-hover-color);
     text-decoration-color: var(--link-hover-color);
 }
 
-.details-card .card-header button:focus h5{
+.details-card .card-header button:focus {
+    outline:none;
```

Comment:
Couldn't you put this into `.details-card .card-header button` or are there some states that we want to have an outline?